src/platforms/android/configuration/integrations/room-and-sqlite.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Room and SQLite
+caseStyle: camelCase
+supportLevel: production
+categories:
+  - mobile
+---
+
+<Note>
+
+Supported in Sentry's Android SDK version `4.0.0` and above.
+
+Supported in Sentry Android Gradle Plugin version `3.0.0` and above.
+
+</Note>
+
+The [Sentry Android Gradle Plugin](/platforms/android/gradle/) provides Room and AndroidX SQLite support through bytecode manipulation. The source can be found [on GitHub](https://github.com/getsentry/sentry-android-gradle-plugin/tree/main/plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation).
+
+On this page, we get you up and running with Sentry's Room and SQLite Integration, so that it will automatically start a span out of the active transaction bound to the scope for each sqlite/dao query.
+
+## Install
+
+To use the Room and AndroidX SQLite integration add the Sentry Android Gradle plugin and the Sentry Android SDK (version `4.0.0` or above) in `build.gradle`:
+
+```groovy
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id "io.sentry.android.gradle" version "3.0.0-beta.3"
+}
+
+dependencies {
+  implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '5.0.0') }}'
+}
+```
+
+```kotlin
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("io.sentry.android.gradle") version "3.0.0-beta.3"
+}
+
+dependencies {
+  implementation("io.sentry:sentry-android:{{ packages.version('sentry.java.android', '5.0.0') }}")
+}
+```
+
+<Note>
+
+Make sure, that [performance monitoring](/platforms/android/performance/#configure-the-sample-rate) is enabled.
+
+</Note>
+
+## Configure
+
+In general, no further configuration required as the auto-instrumentation is enabled by default. If you would like to disable the database instrumentation feature, we expose a configuration option for that:
+
+```groovy
+import io.sentry.android.gradle.InstrumentationFeature
+
+sentry {
+  tracingInstrumentation {
+    enabled = true
+    features = EnumSet.allOf(InstrumentationFeature) - InstrumentationFeature.DATABASE
+  }
+}
+```
+
+```kotlin
+import java.util.EnumSet
+import io.sentry.android.gradle.InstrumentationFeature
+
+sentry {
+  tracingInstrumentation {
+    enabled.set(true)
+    features.set(EnumSet.allOf(InstrumentationFeature::class.java) - InstrumentationFeature.DATABASE)
```

Comment:
Could it be just `setOf(...)`?

Reply:
It could, but you'd need to state all InstrumentationFeature's except the DATABASE. In case we have the number of features growing, this will become too verbose.

Reply:
So I'd leave it like this, this is easier when you want to exclude just one feature I believe